modules/gapi/perf/common/gapi_render_perf_tests.cpp

```diff
@@ -0,0 +1,1238 @@
+                                Values(0),
+                                Values(CV_8UC3),
+                                Values(AbsExact().to_compare_f())));
+
+INSTANTIATE_TEST_CASE_P(RenderTestCircles, RenderTestCircles,
+                        Combine(Values(szVGA, sz720p, sz1080p),
+                                Values(cv::Point(100, 100)),
+                                Values(10),
+                                Values(cv::Scalar(100, 50, 150)),
+                                Values(2),
+                                Values(LINE_8),
+                                Values(0),
+                                Values(CV_8UC3)));
+
+INSTANTIATE_TEST_CASE_P(RenderTestLines, RenderTestLines,
+                        Combine(Values(szVGA, sz720p, sz1080p),
+                                Values(cv::Point(100, 100)),
+                                Values(cv::Point(200, 200)),
+                                Values(cv::Scalar(100, 50, 150)),
+                                Values(2),
+                                Values(LINE_8),
+                                Values(0),
+                                Values(CV_8UC3)));
+
+INSTANTIATE_TEST_CASE_P(RenderTestMosaics, RenderTestMosaics,
+                        Combine(Values(szVGA, sz720p, sz1080p),
+                                Values(cv::Rect(100, 100, 200, 200)),
+                                Values(25),
+                                Values(0),
+                                Values(CV_8UC3)));
+
+INSTANTIATE_TEST_CASE_P(RenderTestImages, RenderTestImages,
+                        Combine(Values(szVGA, sz720p, sz1080p),
+                                Values(cv::Rect(50, 50, 100, 100)),
+                                Values(cv::Scalar(100, 150, 60)),
+                                Values(1.0),
+                                Values(CV_8UC3)));
+
+INSTANTIATE_TEST_CASE_P(RenderTestPolylines, RenderTestPolylines,
+                        Combine(Values(szVGA, sz720p, sz1080p),
+                                Values(std::vector<cv::Point>{{100, 100}, {200, 200}, {150, 300}, {400, 150}}),
+                                Values(cv::Scalar(100, 150, 60)),
+                                Values(2),
+                                Values(LINE_8),
+                                Values(0),
+                                Values(CV_8UC3)));
+
+INSTANTIATE_TEST_CASE_P(RenderTestPolyItems, RenderTestPolyItems,
+                        Combine(Values(szVGA, sz720p, sz1080p),
+                                Values(50),
+                                Values(50),
+                                Values(50)));
+
+
+#ifdef HAVE_FREETYPE
+PERF_TEST_P_(RenderTestFTexts, RenderFTextsPerformanceBGROCVTest)
+{
+    std::wstring text;
+    cv::Size sz;
+    cv::Point org;
+    int fh;
+    cv::Scalar color;
+    MatType type;
+
+    std::tie(text ,sz ,org ,fh ,color, type) = GetParam();
+
+    cv::Mat gapi_mat = cv::Mat(sz, type);
```

Comment:
Why don't you use `create_rand_mats` here

Reply:
Fixed